### PR TITLE
06/07develop

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
# Issue Tracking

JIRA: FWF 686, FWF 716, Footer Name change
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-api/src/api/utils/constants.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
 FORMSFLOW_API_CORS_ORIGINS = os.getenv("FORMSFLOW_API_CORS_ORIGINS")
+<<<<<<< HEAD
+if FORMSFLOW_API_CORS_ORIGINS == "*":
+    CORS_ORIGINS = "*"
+else:
+=======
 if FORMSFLOW_API_CORS_ORIGINS=="*": CORS_ORIGINS = "*"
 else: 
+>>>>>>> d6193976573a24f5bb6f02b70b1dbe2e246f0a3d
```
Comment (abhilash-aot) [inline forms-flow-api/src/api/utils/constants.py:11]: Why is there conflict?

**Review by abhilash-aot — CHANGES_REQUESTED** [review]

File: forms-flow-api/src/api/utils/constants.py (hunk 1 above)
Comment (kurianbenoy-aot) [reply forms-flow-api/src/api/utils/constants.py]: Resolved

**Review by sumathi-thirumani-aot — APPROVED** [review]

**Review by sumathi-thirumani-aot — CHANGES_REQUESTED** [review]
@shibin-aot Please remove the global import of packages

**Comment by shibin-aot** [comment]
The changes will be taken later, since the intention of the review comment is not clear. As this change doesn't impact any functionality nor the performance let us make this change later.

**Review by abhilash-aot — APPROVED** [review]

**Review by shibin-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/BPMFormDataPipelineListener.java [hunk 2]
```diff
@@ -7,7 +7,11 @@
 import lombok.NoArgsConstructor;
 import org.apache.commons.collections.MapUtils;
 import org.apache.commons.lang3.StringUtils;
-import org.camunda.bpm.engine.delegate.*;
+import org.camunda.bpm.engine.delegate.DelegateExecution;
```
Comment (kurianbenoy-aot) [inline forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/BPMFormDataPipelineListener.java:10]: The requested changes have been made by @shibin-aot

**Pull request merged**